api/middlewares/upload_middleware.go

```diff
@@ -104,6 +109,12 @@ func (cfg UploadConfig) ThumbEnable(enable bool) UploadConfig {
 	return cfg
 }
 
+// WEBpEnabled enable thumbnail generation
+func (cfg UploadConfig) Webp(enable bool) UploadConfig {
+	cfg.WebpEnabled = enable
+	return cfg
+}
+
```

Comment:
@Denes-cilwal I want you to change the name of `Webp` to `WebpEnable` to keep it consistent with the previous properties setters. 🙇